tests/bwc/test_rolling_upgrade.py

```diff
@@ -45,7 +45,6 @@
     UpgradePath('5.9.x', '5.10.x'),
     UpgradePath('5.10.x', '6.0.x'),
     UpgradePath('6.0.x', '6.0'),
-    UpgradePath('6.0', '6.1.x'),
```

Comment:
We should keep testing the `6.1` branch, afaik all issues are merged into it already.
```suggestion
    UpgradePath('6.0', '6.1'),
```

The following path 6.1.0 -> 6.1 should probably be removed instead.